phpunit/experimental/interactivity-api/directive-processing-test.php

```diff
@@ -174,42 +214,36 @@ public function test_evaluate_function_should_return_null_for_unresolved_paths()
 
 	public function test_evaluate_function_should_execute_anonymous_functions() {
 		$context = new WP_Directive_Context( array( 'count' => 2 ) );
-		$helper  = new Helper_Class();
 
 		wp_store(
 			array(
 				'state'     => array(
 					'count' => 3,
 				),
 				'selectors' => array(
-					'anonymous_function'           => function ( $store ) {
+					'anonymous_function'  => function ( $store ) {
 						return $store['state']['count'] + $store['context']['count'];
 					},
 					// Other types of callables should not be executed.
-					'function_name'                => 'gutenberg_test_process_directives_helper_increment',
-					'class_method'                 => array( $helper, 'increment' ),
-					'class_static_method'          => 'Helper_Class::static_increment',
-					'class_static_method_as_array' => array( 'Helper_Class', 'static_increment' ),
+					'function_name'       => array( $this, 'gutenberg_test_process_directives_helper_increment' ),
+					'class_method'        => array( $this, 'increment' ),
+					'class_static_method' => array( $this, 'static_increment' ),
```

Comment:
I'm curious: what was the reason behind this change? 🤔

Asking because the idea here was to test different types of [callables](https://www.php.net/manual/en/language.types.callable.php). The way they are defined matters, I'm afraid.

Reply:
I have reverted the changes and fixed the test in https://github.com/WordPress/gutenberg/pull/56302/commits/ba0d2374f22436b1c0495e8d1af9901cd0e90fde.